userspace/libsinsp/parsers.cpp

```diff
@@ -6084,4 +6087,56 @@ void sinsp_parser::parse_memfd_create_exit(sinsp_evt *evt, scap_fd_type type)
 	}
 
 	evt->m_fdinfo = evt->m_tinfo->add_fd(fd, &fdi);
+}
+
+void sinsp_parser::parse_pidfd_getfd_exit(sinsp_evt *evt)
+{
+
+	sinsp_evt_param* parinfo;
+	int64_t fd;
+	int64_t pidfd;
+	int64_t targetfd;
+
+	ASSERT(evt->m_tinfo)
+	if(evt->m_tinfo == nullptr)
+	{
+		return;
+	}
+
+	/* ret (fd) */
+	parinfo = evt->get_param(0);
+	ASSERT(parinfo->m_len == sizeof(int64_t));
+	ASSERT(evt->get_param_info(0)->type == PT_FD);
+	fd = *(int64_t *)parinfo->m_val;
+
+	/* pidfd */
+	parinfo = evt->get_param(1);
+	ASSERT(parinfo->m_len == sizeof(int64_t));
+	ASSERT(evt->get_param_info(1)->type == PT_FD);
+	pidfd = *(int64_t *)parinfo->m_val;
+
+	auto target_tinfo = m_inspector->get_thread_ref(pidfd);
```

Comment:
I think this is wrong: pidfd is an fd obtained with `pidfd_open`, right?
You are trying to use it as a pid, while it is a pidfd instead.

Reply:
Yes, you are right!!. So first I need to implement `pidfd_open` syscall and then update the parsers

Reply:
Will close this for now!!